tap_pendo/streams.py

```diff
@@ -406,7 +406,9 @@ def sync_substream(self, state, parent, sub_stream, parent_response):
                             integer_datetime_fmt=
                             "unix-milliseconds-integer-datetime-parsing"
                         ) as transformer:
-                    stream_events = sub_stream.sync(state, new_bookmark,
+                    # bug fix for syncing child streams from start date or
```

Comment:
@savan-chovatiya Instead of writing 'bug fix', you can update the comment to say:
"syncing child streams from start date or state file date"